implementations/kmp-json-schema-validator/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM gradle:8.2.1-jdk11
+WORKDIR /opt/app
+COPY gradle.properties .
+COPY settings.gradle.kts .
+COPY build.gradle.kts .
+RUN gradle dependencies > /dev/null
```

Comment:
What this line do? Does it have some effect?

Reply:
Yes, it is used to initialize some part of the Gradle caches and reuse it in future builds.

Docker caches each step. So, if nothing is changed in gradle.properties, settings.gradle.kts or build.gradle.kts the docker starts right from the cached image that was produced by this step. In this case the command `RUN gradle install` will be executed faster (at least it is my understanding of that and it seems to be working on my machine)